experiments/integrated/fluxnet/met_drivers_FLUXNET.jl

```diff
@@ -0,0 +1,142 @@
+using ArtifactWrappers
+using DelimitedFiles
+using Dierckx
+using Thermodynamics
+using Dates
+using Formatting
+
+af = ArtifactFile(
+    url = data_link,
+    filename = "AMF_$(site_ID)_FLUXNET_FULLSET.csv",
+)
+dataset = ArtifactWrapper(@__DIR__, "ameriflux_data", ArtifactFile[af]);
+dataset_path = get_data_folder(dataset);
+data = joinpath(dataset_path, "AMF_$(site_ID)_FLUXNET_FULLSET.csv");
+driver_data = readdlm(data, ',')
+
+# Label of every data column to be collected from the driver data file
+labels = (:TA, :VPD, :PA, :P, :WS, :LW_IN, :SW_IN, :G, :GPP, :LE, :H, :LW_OUT,
+          :SW_OUT, :SWC, :TS, :CO2, :LAI)
+
+# For every data column to be collected, name of the column in the file, 
+# transformation function to desired unit, and string of final unit
+collect_args = [("TA_F", (x) -> x .+ 273.15, "K")
+               ("VPD_F", (x) -> x .* 100, "Pa")
+               ("PA_F", (x) -> x .* 1000, "Pa")
+               ("P_F", (x) -> x ./ (1000 * 1800), "m/s")
```

Comment:
I think we need to be careful with precip, because if the fluxnet data is HH vs H this conversion changes

Reply:
is snow precip available?

Reply:
I don't believe the fluxnet data has snow precip, at least it's not listed [here](https://fluxnet.org/data/fluxnet2015-dataset/subset-data-product/)

Reply:
Great catch on the timestep for the precip data

Reply:
@AlexisRenchon do fluxnet sites record snow precip separately from liquid water precip?

Reply:
I am pretty sure most sites have a "Precip" data which includes rain and snow in mm of water

Reply:
but it seems some site may have rain and snow separately, see here https://fluxnet.org/data/aboutdata/data-variables/ 
<html>
<body>
<!--StartFragment-->
MET_PRECIP |   |  
-- | -- | --
P | mm | Precipitation
P_RAIN | mm | Rainfall
P_SNOW | mm | Snowfall
D_SNOW | cm | Snow depth
RUNOFF | mm | Run off

<!--EndFragment-->
</body>
</html>

Reply:
ok! thanks. When we add snow to the model, we can deal with it!